raster-foundry/raster-foundry
# Remove tile subproject and nginx config

## Overview

This pr removes the tile server subproject, nginx configuration for the tile server, tile server references from docker-compose files, and artifact publication from scripts/cipublish. It also defaults the local server strategy to the `assembly` and `docker-compose.java.yml`, since we never figured out what the issue is with recompiling all the way back to bridge, it's still a problem, and `--backsplash` is sort of a meaningless flag when there's no alternative. 

### Checklist


## Testing Instructions

 * make sure you have recently `assembly`-ed backsplash and api-server jars
 * `./scripts/server`
 * use the app
 * confirm there are no tile-server or nginx-tiler references in docker-compose*.yml files
 * confirm the tile server is gone from `nginx/`
 * confirm backsplash-core and backsplash-server are getting compiled in cibuild

Closes #4469
